Superposition/Superposition.ipynb

```diff
@@ -839,7 +850,7 @@
    "file_extension": ".qs",
    "mimetype": "text/x-qsharp",
    "name": "qsharp",
-   "version": "0.14"
+   "version": "0.24"
```

Comment:
It's not advisable to change the versions in Jupiter notebook when checking it in git. These changes are done automatically by jupyter when we run notebooks locally. Before checking in notebook, I usually use "git add -patch" to check in only the desired changes


```suggestion
   "version": "0.14"
```